resources/css/landingpage.css

```diff
@@ -503,19 +503,29 @@ nav {
 .row1 {
     margin-top: 20px;
     width: 30%;
-    display: grid;
-    grid-template-columns: 1fr 8fr;
-    grid-template-rows: 1fr 1fr 1fr;
     margin-left: 60px;
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
```

Comment:
you have to maintain overall gap of 36 b/w icons, since you have used 16 as margin, adjust it accordingly?